builds/e2e/e2e-release.yaml

```diff
@@ -0,0 +1,170 @@
+parameters:

```

Comment:
This seems very similar to e2e.yaml . Can we combine them?

Reply:
You're right, they are indeed very similar.

Here's where they're different:

1) They call different templates: e2e-setup-base-image-update-release.yaml is sufficiently different to e2e-setup.yaml
2) There's a dependency on PublishManifests that doesn't exist in e2e.yaml: _builtImages: $[ stageDependencies.PublishManifests.PublishManifest.result ]_
3) e2e.yaml also runs _centos7_amd64_ & _linux_amd64_proxy_ (which, as you know, are not Tier 1)

I should add that till such time this pipeline has truly earned our trust - by delivering the right things across a couple of releases - we should look at it with suspicion. We definitely don't want to conflate it with the existing release process and break existing workflows.

Reply:
> 3. e2e.yaml also runs _centos7_amd64_ & _linux_amd64_proxy_ (which, as you know, are not Tier 1)

Linux amd64 behind a proxy _is_ Tier 1 (rather, Ubuntu is Tier 1 and IoT Edge behind a proxy is supported). But I agree that we should keep this pipeline separate to start, then look at how best to refactor once it has settled a bit.